helper-cli/src/main/kotlin/commands/ExtractRepositoryConfigurationCommand.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
```

Comment:
@fviernau, @mnonnenmacher, so what do we do about the year range for HERE copyright statements in new files now? The recent PRs introduced a mix of "Copyright (C) 2017-2019 HERE Europe B.V." and "Copyright (C) 2019 HERE Europe B.V.".

Reply:
discussed with @sschuberth, @mnonnenmacher, agreed on 2017-2019.